### PR TITLE
Add event frequency function

### PR DESCRIPTION
**Pull request by ogarocious**

### CODE REVIEW

File: app/views/artists/show.html.erb [hunk 1]
```diff
@@ -15,8 +33,36 @@
...
+
+</div>
+
+<div class="";>
+<h3>Current Booked Events</h3>
+  <% if @artist.events.any? %>
+=======
   <div class="artist-video-wrapper">
       <h3>Artist Videos</h3>
     <div class="row row-cols-4">
```
Comment (melchua) [inline app/views/artists/show.html.erb]: there's a merge conflict here you should address

File: app/views/artists/show.html.erb [hunk 2]
```diff
@@ -58,4 +105,21 @@
   </div>
 </div>
 
+<<<<<<< HEAD
```
Comment (melchua) [inline app/views/artists/show.html.erb]: merge conflict to address

**Review by melchua — CHANGES_REQUESTED** [review]
Looks like there's some merge conflicts to fix before i can do any testing.

File: app/views/artists/show.html.erb [hunk 3]
```diff
@@ -1,4 +1,26 @@
 <div class="show-page-wrapper">
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+>>>>>>> master
```
Comment (melchua) [inline app/views/artists/show.html.erb]: conflict

File: app/views/artists/show.html.erb [hunk 4]
```diff
@@ -1,4 +1,26 @@
...
+    <div class="row row-cols-4">
+      <div class="mt-2">
+        <%= render @artist.dance_styles %>
+      </div>
+    </div>
+  <% end %>
+ <div class="card-body">
+  <%# <h5 class="card-title"><%= @artist.name </h5> %>
+    <%# <p class="card-text"><%= @artist.description </p> %>
+<<<<<<< HEAD
```
Comment (melchua) [inline app/views/artists/show.html.erb]: conflict

File: app/views/artists/show.html.erb [hunk 5]
```diff
@@ -15,8 +37,41 @@
   <%# <h5 class="card-title"><%= @artist.name </h5> %>
     <%# <p class="card-text"><%= @artist.description </p> %>
       </div>
+>>>>>>> master
   </div>
 
+<<<<<<< HEAD
```
Comment (melchua) [inline app/views/artists/show.html.erb]: conflict

File: app/views/artists/show.html.erb [hunk 6]
```diff
@@ -15,8 +37,41 @@
...
+=======
+  </div>
+
+>>>>>>> master
```
Comment (melchua) [inline app/views/artists/show.html.erb]: conflcit

File: app/views/artists/show.html.erb [hunk 7]
```diff
@@ -39,6 +94,9 @@
   <div class="";>
       <h3>Current Booked Events</h3>
        <% if @artist.events.any? %>
+>>>>>>> master
```
Comment (melchua) [inline app/views/artists/show.html.erb]: conflict

File: app/views/artists/show.html.erb [hunk 8]
```diff
@@ -58,4 +116,27 @@
...
+
+      <%= link_to '<button type="button" class="btn btn-outline-danger mt-1">
+      Delete artist
+      </button>'.html_safe, artist_path(@artist), method: :delete, data: { confirm: "Are you sure?"} %>
+    <% end %>
+  </div>
+</div>
+</div>
+</div>
+<<<<<<< HEAD
```
Comment (melchua) [inline app/views/artists/show.html.erb]: conflict

File: app/views/event_frequencies/index.json.jbuilder [hunk 9]
```diff
@@ -0,0 +1 @@
+json.array! @event_frequencies, partial: "event_frequencies/event_frequency", as: :event_frequency
```
Comment (melchua) [inline app/views/event_frequencies/index.json.jbuilder:1]: investigate what this does

File: app/views/artists/show.html.erb (hunk 1 above)
Comment (ogarocious) [reply app/views/artists/show.html.erb]: cleaned up

File: app/views/artists/show.html.erb (hunk 2 above)
Comment (ogarocious) [reply app/views/artists/show.html.erb]: cleaned up

File: app/views/artists/show.html.erb (hunk 3 above)
Comment (ogarocious) [reply app/views/artists/show.html.erb]: cleaned up

File: app/views/artists/show.html.erb (hunk 4 above)
Comment (ogarocious) [reply app/views/artists/show.html.erb]: cleaned up

File: app/views/artists/show.html.erb (hunk 5 above)
Comment (ogarocious) [reply app/views/artists/show.html.erb]: cleaned up

File: app/views/artists/show.html.erb (hunk 6 above)
Comment (ogarocious) [reply app/views/artists/show.html.erb]: cleaned up

File: app/views/artists/show.html.erb (hunk 7 above)
Comment (ogarocious) [reply app/views/artists/show.html.erb]: cleaned up

File: app/views/artists/show.html.erb (hunk 8 above)
Comment (ogarocious) [reply app/views/artists/show.html.erb]: cleaned up

File: app/views/event_frequencies/index.json.jbuilder (hunk 9 above)
Comment (ogarocious) [reply app/views/event_frequencies/index.json.jbuilder:1]: I believe this was a bit of rails magic that was created when I created the event frequencies scaffold

**Review by melchua — APPROVED** [review]

**Pull request merged**